.gitignore

```diff
@@ -0,0 +1,12 @@
+*.pyc
+
+# OS generated files #
+######################
+.DS_Store
+.DS_Store?
+._*
+.Spotlight-V100
+.Trashes
+ehthumbs.db
+Thumbs.db
+
```

Comment:
perhaps you should have added *~ and eclcc.log and *.odt ?

Reply:
I can get rid of eclcc.log.  That was a mistake.  Documentation.odt is the main documentation for the project.